packages/example/src/components/Transfer/Transfer.tsx

```diff
@@ -80,17 +79,21 @@ export const Transfer: React.FC<ITransferProps> = ({network}) => {
         if(provider && provider.signer.signPayload) {
             if (amount && recipient) {
                 const api = await provider.getMetamaskSnapApi();
-                const txPayload = await api.generateTransactionPayload(amount, recipient);
+                const amountWND = BigInt(amount) * BigInt("1000000000");
```

Comment:
@mpetrunic This is not a solution, I am changing this before moving from draft state of PR.